setup.py

```diff
@@ -216,12 +223,14 @@ def members_appended(*ds):
 
     if cythonize and cythonizing:
         cython_opts = dict(
-            # compile .pyx extensions to .c in parallel
-            nthreads=cpu_threads + 1,
             # default language_level will be '3str' starting from Cython 3.0.0,
             # but old cython versions (< 0.29) do not know that, thus we use 3 for now.
             compiler_directives={'language_level': 3},
         )
+        if os.name != 'nt':
```

Comment:
above you use sys.platform, here you use os.name - accidental?

maybe there could be a 1-liner at the top of setup.py, like:
```
is_win32 = sys.platform.startswith('win32')
```
(taken from borg.platformflags)